src/main/kotlin/coffeebot/commands/Milton.kt

```diff
@@ -27,30 +28,34 @@ class MiltonClient {
     }
     private val miltonHost = "https://milton.terbium.io"
 
-    fun index(url: URL): Boolean {
+    private fun index(url: URL): Boolean {
         val result = runBlocking(Dispatchers.IO) {
-            httpClient.post<HttpResponse>("$miltonHost/save?url=${url}")
+            httpClient.post<HttpResponse>("$miltonHost/save?url=${url}") {
+                header("Authorization", "Bearer bot;coffeebot;$secret")
+            }
         }
+        println(result)
         return result.status == HttpStatusCode.OK
     }
-}
-
-private val miltonClient = MiltonClient()
-private val urlRegex = "https?://[\\w\\d:#@%/;$()~_?+-=\\\\.&]*".toRegex(RegexOption.IGNORE_CASE)
 
-val milton = PassiveCommand { message ->
-    val matches = urlRegex.findAll(message.contents).toList()
-    val results = matches.map { match ->
-        val urlValue = match.value
-        val url = try {
-            URL(urlValue)
-        } catch (e: MalformedURLException) {
-            throw IllegalStateException("invalid url: $urlRegex")
+    val command = PassiveCommand { message ->
+        val matches = urlRegex.findAll(message.contents).toList()
+        val results = matches.map { match ->
+            val urlValue = match.value
+            val url = try {
+                URL(urlValue)
+            } catch (e: MalformedURLException) {
+                throw IllegalStateException("invalid url: $urlRegex")
+            }
+            index(url)
+        }
+        if (matches.isNotEmpty()) {
+            val emoji = if (results.all { it }) "\u2705" else "\u274E"
```

Comment:
I know this code has already been committed, but: 

1) Is it possible to insert the emoji directly in the source?

2) Could you add a comment about what results.all represents (that all urls were indexed successfully by Milton)

Reply:
Added the emojis; I've added a doc comment to the `index` method because I thought that made more sense than adding one to the `results` variables, let me know if you disagree.